7.5.цикл_while:_обработка_цифр_числа/7.5.4.py

```diff
@@ -0,0 +1,5 @@
+n = int(input())
+while n != 0:  
+    l = n % 10  
+    n = n // 10 
```

Comment:
Не забываем использовать расширенные операторы присваивания:
```suggestion
    n //= 10 
```